response_operations_ui/static/js/data-panel.js

```diff
@@ -0,0 +1,84 @@
+function openPanel(){
```

Comment:
possibly missing a param and requires logic changes? can't see how this functions identifies which panel to show

Reply:
This was just copy/pasted across from the dropbox, I'll look into cleaning it up today